intuita-webview/src/codemodList/TreeView/index.tsx

```diff
@@ -44,6 +49,25 @@ const TreeView = ({
 	const [focusedNodeId, setFocusedNodeId] = useState('');
 	const [editExecutionPath, setEditExecutionPath] =
 		useState<CodemodTreeNode<string> | null>(null);
+	const [progress, { progressBar, stopProgress }] = useProgressBar();
+	const [executionStack, setExecutionStack] = useState<Set<CodemodHash>>(
+		new Set(),
+	);
+
+	useEffect(() => {
+		if (progress || executionStack.size === 0) {
+			return;
+		}
+		const hash = Array.from(executionStack).shift();
```

Comment:
if we use set, we should use the alphanumerical sorting to have this line always yield the same results